src/ocaml_specific.ml

```diff
@@ -693,13 +693,19 @@ let () =
       ["c"; "compile"];
     ] in
 
-    (* tags package(X), predicate(X) and syntax(X) *)
+    (* tags package(X), predicate(X) and syntax(X), ppopt(X), ppxopt(X) *)
     List.iter begin fun tags ->
       pflag tags "package" ~doc_param (fun pkg -> S [A "-package"; A pkg]);
       pflag tags "predicate" ~doc_param:"archive"
         (fun pkg -> S [A "-predicates"; A pkg]);
-      if List.mem "ocaml" tags then
-        pflag tags "syntax" ~doc_param:"camlp4o" (fun pkg -> S [A "-syntax"; A pkg])
+      if List.mem "ocaml" tags then begin
+        pflag tags "syntax" ~doc_param:"camlp4o"
+          (fun pkg -> S [A "-syntax"; A pkg]);
+        pflag tags "ppopt" ~doc_param:"camlp4-arg"
```

Comment:
-pp is not just camlp4, isn't it? ocaml-m17n also uses -pp.

Reply:
I reused the name from the `ocamlfind` documentation, and it is just informative (there is no semantics on the `doc_param`). Yet your point is completely correct and I will change it to use `pparg` instead.